package.json

```diff
@@ -26,10 +26,8 @@
   },
   "dependencies": {
     "bson": "^5.4.0",
-    "mongodb-connection-string-url": "^2.6.0"
-  },
-  "optionalDependencies": {
-    "saslprep": "^1.0.3"
+    "mongodb-connection-string-url": "^2.6.0",
+    "@mongodb-js/saslprep": "^1.1.0"
```

Comment:
(Unrelated just picking a line)

Just some highlight clarifying suggestions, I think it would be nice to complete the picture on what actually breaks:

> breaks when bundled with webpack

"breaks when bundled with webpack because it attempted to read a file relative to the package location"

> that can be bundled with webpack

"that can be bundled with webpack because it includes the necessary saslprep data in memory upon loading"